db/warehouse/migrate/20230927205059_add_service_indexes.rb

```diff
@@ -0,0 +1,10 @@
+class AddServiceIndexes < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      add_index :CustomServices, :EnrollmentID
+      add_index :CustomServices, :PersonalID
+      add_index :CustomServices, :data_source_id
+      add_index :CustomServices, :DateProvided
```

Comment:
It sounds like this provided some performance gains, so probably fine as is, but I wonder if we might want some compound indexes.  The `data_source_id` index specifically is probably doing very little since I believe all CustomServices would be in the same data source (at least for now.) The others may be used exclusively, looking for the services for a particular enrollment, or a particular client, or on a particular day, but if they are used in conjunction (does enrollment always include a client?) then we might want `[:EnrollmentID, :PersonalID]`

Reply:
Thanks, let's revisit this after launch